docs/en/02_Developer_Guides/05_Extending/05_Injector.md

```diff
@@ -141,7 +141,33 @@ SilverStripe\Core\Injector\Injector:
       - [ pushHandler, [ %$DefaultHandler ] ]
 ```
 
-## Using constants as variables
+[info]
```

Comment:
Not sure what the value of putting this in an '[info]' block is. Otherwise, it's a great contribution.

Reply:
thought it was more of an 'aside' but happy to take it out :)